cstore_fdw.h

```diff
@@ -272,21 +282,23 @@ typedef struct TableReadState
 	uint64 stripeReadRowCount;
 	ColumnBlockData **blockDataArray;
 	int32 deserializedBlockIndex;
+	Relation relation;
 
 } TableReadState;
 
 
 /* TableWriteState represents state of a cstore file write operation. */
 typedef struct TableWriteState
 {
-	FILE *tableFile;
 	TableFooter *tableFooter;
-	StringInfo tableFooterFilename;
 	CompressionType compressionType;
+	bool logging;
 	TupleDesc tupleDescriptor;
 	FmgrInfo **comparisonFunctionArray;
 	uint64 currentFileOffset;
 	Relation relation;
+	BlockNumber activeBlockNumber;
+
```

Comment:
I will have to run citus_indent on all files. It is generating far more changes than we want to review in the context of this pr.